pyproject.toml

```diff
@@ -5,3 +5,15 @@ exclude = '''
     \.git
 )/
 '''
+
+[tool.isort]
+skip = ['.git', 'venv']
+known_tests = ['tests']
+sections = ['FUTURE', 'STDLIB', 'THIRDPARTY', 'FIRSTPARTY', 'TESTS', 'LOCALFOLDER']
+default_section = 'THIRDPARTY'
+include_trailing_comma = true
+force_grid_wrap = 0
+use_parentheses = true
+multi_line_output = 3
+line_length = 78
+float_to_top = true
```

Comment:
This is a bit different from what is in `closeio`. Is that intentional? Line length is 79 in `closeio`, and we also have `combine_as_imports = true`.

Reply:
Sorry for not flagging this as part of my initial review, I only had the idea to compare them after I had already submitted. 😬

Reply:
That's a good point. I actually scavenged and revised the config from a repository that has a slightly different config than our ones. I'll open another PR to bring this one in sync.